### DISCUSSION TITLE
3.6.1rc0

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1344] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.6.1rc0'>3.6.1rc0</a>.</em><hr />

Release Candidate 0 Status:

- [x] Linux ARM wheels (https://github.com/pyproj4/pyproj-wheels/pull/58)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/140, https://github.com/conda-forge/pyproj-feedstock/pull/141)
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Plan to release in a few days. Testing is always appreciated, but definitely not required.

**Comment by sebastic** [comment]
Debian package build looks good:

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental

**Reply by snowman2** [reply]
Thanks @sebastic 👍

**Comment by jdkloe** [comment]
Manual build in Fedora rawhide mock environment looks good as well. Compiles fine and all tests pass.
Software versions used for testing:
Python 3.12.0rc2
Cython-3.0.2
gcc-13.2.1
proj-9.3.0

**Reply by snowman2** [reply]
Thank @jdkloe 👍
